plugins/UsersManager/API.php

```diff
@@ -698,12 +698,14 @@ public function addUser($userLogin, $password, $email, $alias = false, $_isPassw
      * @param string   $userLogin          the user login.
      * @param bool|int $hasSuperUserAccess true or '1' to grant Super User access, false or '0' to remove Super User
      *                                     access.
+     * @param string $passwordConfirmation the current user's password.
      * @throws \Exception
      */
-    public function setSuperUserAccess($userLogin, $hasSuperUserAccess)
+    public function setSuperUserAccess($userLogin, $hasSuperUserAccess, $passwordConfirmation)
```

Comment:
I guess we should mention that *breaking* API change in the changelog

Reply:
Added to the changelog